### PR TITLE
Make friendly doggies nicer.

### PR DESCRIPTION
**Pull request by alexjurkiewicz**

### CODE REVIEW

File: crawl-ref/source/dat/database/monspeak.txt [hunk 1]
```diff
@@ -842,11 +842,34 @@ friendly hound
...
+VISUAL:@The_monster@ looks up at you expectantly.
+
+VISUAL:@The_monster@ scratches @possessive@ ear.
+
+VISUAL:@The_monster@ cocks @possessive@ head sideways at you.
+
+VISUAL:@The_monster@ scratches at a flea.
+
+w:1
+VISUAL:@The_monster@ gnaws at a bone -- where's that from?
```
Comment (wheals) [inline crawl-ref/source/dat/database/monspeak.txt]: should probably be an em dash

**Comment by alexjurkiewicz** [comment]
updated as per @wheals

File: crawl-ref/source/dat/database/monspeak.txt [hunk 2]
```diff
@@ -842,11 +842,34 @@ friendly hound
...
+VISUAL:@The_monster@ cocks @possessive@ head sideways at you.
+
+VISUAL:@The_monster@ scratches at a flea.
+
+w:1
+VISUAL:@The_monster@ gnaws at a bone — where's that from?
+
 @hound@
 %%%%
-default insect
```
Comment (wheals) [inline crawl-ref/source/dat/database/monspeak.txt]: This probably isn't supposed to remove `default insect` altogether?

File: crawl-ref/source/dat/database/monspeak.txt [hunk 3]
```diff
@@ -842,11 +842,34 @@ friendly hound
...
+friendly hound Shoals
 
-VISUAL:@The_monster@ waves @possessive@ antennae.
+w:50
+@friendly hound@
+
+VISUAL:@The_monster@ [plays|frolics] in the surf!
+
+There's a distinct wet dog smell emanating from @The_monster@.
```
Comment (wheals) [inline crawl-ref/source/dat/database/monspeak.txt]: should this be `@the_monster@`?

Comment (alexjurkiewicz) [inline crawl-ref/source/dat/database/monspeak.txt]: derp. fixed

**Comment by wheals** [comment]
None of these look too distracting, except maybe the very low-weighted ones, which seems fine to me.

**Pull request merged**